lib/schemas/nsm/v1.json

```diff
@@ -1271,6 +1272,14 @@
       "x-pii": false,
       "type":"string"
     },
+    "main_offence_type":{
+      "examples":[
+        "Summary only"
```

Comment:
Yes, you probably want to add:

```
"enum": ["summary_only", "either_way", "indictable_only", "prescribed_proceedings"]
```

To enforce that only a valid value gets put here.

And for bonus points, you'd add user-facing translations of those values to the locale files in the gem so that provider and caseworker can share the translation rather than having to duplicate them across codebases.

Reply:
Ah it does already handle it, great.

I'll leave the note in anyway so we can decide if we want to handle these in schema or in code (though I think the former is definitely preferable)